Source/Common/SchemaSerializer.cs

```diff
@@ -13,50 +13,43 @@ public static class SchemaSerializer
-        /// Default serializer settings used when deserializing
-        /// </summary>
-        private static readonly JsonSerializerSettings DeserializeSettings = new()
-        {
-            DateParseHandling = DateParseHandling.None,
-        };
-
         /// <summary>
         /// Default serializer settings used when HTML escaping is not required.
         /// </summary>
-        private static readonly JsonSerializerSettings DefaultSerializationSettings = new()
-        {
-            Converters = new List<JsonConverter>()
-            {
-                new StringEnumConverter(),
-            },
-            DefaultValueHandling = DefaultValueHandling.Ignore,
-            NullValueHandling = NullValueHandling.Ignore,
-        };
+        private static readonly JsonSerializerOptions DefaultSerializationSettings;
 
         /// <summary>
         /// Serializer settings used when trying to avoid XSS vulnerabilities where user-supplied data is used
         /// and the output of the serialization is embedded into a web page raw.
         /// </summary>
-        private static readonly JsonSerializerSettings HtmlEscapedSerializationSettings = new()
+        private static readonly JsonSerializerOptions HtmlEscapedSerializationSettings;
+
+#pragma warning disable CA1810 // Initialize reference type static fields inline
+        static SchemaSerializer()
+#pragma warning restore CA1810 // Initialize reference type static fields inline
         {
-            Converters = new List<JsonConverter>()
+            DefaultSerializationSettings = new JsonSerializerOptions
             {
-                new StringEnumConverter(),
-            },
-            DefaultValueHandling = DefaultValueHandling.Ignore,
-            NullValueHandling = NullValueHandling.Ignore,
-            StringEscapeHandling = StringEscapeHandling.EscapeHtml,
-        };
+                AllowTrailingCommas = true,
+                DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingDefault,
+                Encoder = System.Text.Encodings.Web.JavaScriptEncoder.UnsafeRelaxedJsonEscaping,
+            };
+
+            HtmlEscapedSerializationSettings = new JsonSerializerOptions
+            {
+                AllowTrailingCommas = true,
+                DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingDefault,
+            };
+        }
 
         /// <summary>
         /// Deserializes the JSON to the specified type.
         /// </summary>
         /// <typeparam name="T">Deserialization target type</typeparam>
         /// <param name="value">JSON to deserialize</param>
         /// <returns>An instance of <typeparamref name="T"/> deserialized from JSON</returns>
-        public static T? DeserializeObject<T>(string value) =>
-            JsonConvert.DeserializeObject<T>(value, DeserializeSettings);
+        public static T? DeserializeObject<T>(string value)
+            => JsonSerializer.Deserialize<T>(value, DefaultSerializationSettings);
```

Comment:
In the latest editorconfig changes in VS 16.11 you can decide to have `=>` or `&&` or `||` at the beginning/end of the line. I went with the end, so I think this would create a warning in later versions of VS.

Reply:
I've been very frustrated with `.editorconfig` since its so buggy and has some undocumented settings. I wouldn't be surprised if you didn't see a warning.